src/routes/+layout.svelte

```diff
@@ -1,53 +1,50 @@
 <script>
 	import '../app.postcss';
 
-	import '@fontsource/inter/300.css';
-	import '@fontsource/inter/400.css';
-	import '@fontsource/inter/500.css';
-	import '@fontsource/inter/600.css';
-	import '@fontsource/inter/700.css';
-	import '@fontsource/inter/800.css';
-	import '@fontsource/inter/900.css';
-
-	import GithubLogo from '~icons/radix-icons/github-logo';
-	import DiscordLogo from '~icons/radix-icons/discord-logo';
-
 	import { dev } from '$app/environment';
 	import { inject } from '@vercel/analytics';
 
 	inject({ mode: dev ? 'development' : 'production' });
 </script>
 
 <main class="flex min-h-screen flex-col">
-	<nav class="flex items-center justify-between gap-2 border-b border-zinc-400 px-2 lg:px-4 py-2">
-		<a class="flex items-center gap-2 lg:text-xl font-sans font-bold" href="/">
+	<nav class="flex items-end justify-between px-6 py-4">
+		<a class="flex items-center gap-2 font-sans text-xl" href="/">
 			<img
-				class="w-7 h-7 lg:h-9 lg:w-9 rounded-sm object-contain"
-				src="/logo.svg"
+				class="h-8 w-8 rounded-sm object-contain"
+				src="/radix-svelte.svg"
 				alt="Radix and Svelte logos"
 			/>
-			Radix Svelte
+			<span class="font-bold">Radix</span> Svelte
 		</a>
 
-		<div class="flex items-center gap-2 lg:gap-4">
+		<div class="flex flex-row gap-4">
 			<a
-				href="https://github.com/tglide/radix-svelte"
-				target="_blank"
-				class="hover:opacity-75 lg:text-xl"
+				href="https://github.com/TGlide/radix-svelte"
+				class="underline-offset-2 opacity-75 hover:underline hover:opacity-100 focus:underline active:opacity-75"
+				>GitHub</a
 			>
-				<GithubLogo />
-			</a>
 			<a
-				href="https://discord.com/invite/gQrpPs34xH"
-				target="_blank"
-				class="hover:opacity-75 lg:text-xl"
+				href="/docs/accordion"
+				class="underline-offset-2 opacity-75 hover:underline hover:opacity-100 focus:underline active:opacity-75"
+				>Documentation</a
 			>
-				<DiscordLogo />
-			</a>
-			<a href="/docs/accordion" class="underline hover:opacity-75">Documentation</a>
 		</div>
 	</nav>
 	<div class="flex grow flex-col">
 		<slot />
 	</div>
+	<footer>
+		<div
+			class="flex flex-col items-start justify-between gap-2 px-6 py-4 lg:flex-row lg:items-center"
+		>
+			<span class="flex gap-1">
+				Inspired by <a
+					href="https://radix-ui.com/"
```

Comment:
There's no way to know this is a link without hovering, maybe a dashed underline could work before hover?

Reply:
Yup, the goal is to have a consistent link style similar to the navigation links